lib/noughts_and_crosses.rb

```diff
@@ -30,6 +30,14 @@ def winner
     end.compact.fetch(0, nil)
   end
 
+  def board_full?
```

Comment:
I think you should think more carefully about this name, who cares if the board is full? why? what's the abstraction here?